### PR TITLE
Bring back slot_commitment_id_index test

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/tests/types/slot.rs [hunk 1]
```diff
@@ -1,40 +1,25 @@
...
 use packable::PackableExt;
+use pretty_assertions::assert_eq;
 
-// #[test]
-// fn slot_commitment_id_index() {
-// Test from https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#slot-commitment-id-1
+#[test]
+fn slot_commitment_id_index() {
+    // Test from https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#slot-commitment-id-1
+    let file = std::fs::read_to_string("./tests/types/fixtures/slot_commitment.json").unwrap();
```
Comment (DaughterOfMars) [inline sdk/tests/types/slot.rs]: Add a struct to define the types and then just call https://docs.rs/serde_json/latest/serde_json/fn.from_reader.html?

Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: If we had more than one fixture maybe but here I'm not sure there is any point tbh

Comment (DaughterOfMars) [reply sdk/tests/types/slot.rs]: I mean it's not a lot to add and it helps clarity

Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: Ok, will do

Comment (thibault-martinez) [reply sdk/tests/types/slot.rs]: done

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**